daemon/set.go

```diff
@@ -0,0 +1,57 @@
+package daemon
+
+import (
+	"github.com/docker/docker/runconfig"
+)
+
+func (daemon *Daemon) ContainerSet(name string, hostConfig *runconfig.HostConfig) ([]string, error) {
+	warnings, err := daemon.verifyHostConfig(hostConfig)
+	if err != nil {
+		return warnings, err
+	}
+
+	container, err := daemon.Get(name)
+	if err != nil {
+		return warnings, err
+	}
+
+	if err := daemon.updateResources(container, hostConfig); err != nil {
+		return warnings, err
+	}
+
+	if err := daemon.Set(container.command); err != nil {
+		return warnings, err
+	}
+
+	return warnings, nil
+}
+
+func (daemon *Daemon) updateResources(container *Container, hostConfig *runconfig.HostConfig) error {
```

Comment:
there is nothing returning errors in this function, should we remove the `return` statement?